CONTRIBUTING.md

```diff
@@ -72,10 +74,14 @@ and
 * Do not put whitespace fixes/cleanup and functionality changes in the same commit
 
 # Coding Guidelines
-At the moment there is no established coding guideline, but here are some
-general guidelines:
+We have `clang-format` in place, but not for all files yet.
+This is an incremental process.
+If you're adding new code, adhering to the formatting config is appreciated.
+Formatting breakage in already formatted files will be caught by the CI.
+
+Additional guidelines:
 
 * Don't have end-of-line whitespace
 * Use spaces instead of tabs
-* Stick to the style of the file you're editing
 * Functions and classes must have a [docblock](http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html)
```

Comment:
Is docblock still relevant?

Reply:
We all suck at doing it. Perhaps it's unfair to ask this from contributors, yes.

Reply:
also the link is 404

Reply:
Oh, crap, I was supposed to do that? :-D

Reply:
pushed new text